values.yaml

```diff
@@ -325,43 +325,46 @@ server:
 externalServers:
   # If true, the Helm chart will be configured to talk to the external servers.
   # If setting this to true, you must also set server.enabled to false.
-  # Note that if you are setting client.join property, https.address property is not required.
   enabled: false
 
-  # HTTPS configuration for external servers.
-  # Note: HTTP connections to the servers are
-  # not supported.
-  https:
-    # IP, DNS name, or Cloud auto-join string pointing to the external Consul servers.
-    # Port must be provided separately with the externalServers.https.port property.
-    # Note that if you’re providing the cloud auto-join string and multiple addresses
-    # can be returned, only the first address will be used.
-    # This value is required only if you would like to use
-    # a different server address from the one specified
-    # in the client.join property.
-    address: null
-
-    # The HTTPS port of the server.
-    port: 443
+  # An array of external Consul hosts.
+  # Valid values include IPs, DNS names, or Cloud auto-join string.
+  # Ports must be provided separately below.
+  # These hosts are used for clients to join the server cluster,
+  # as well as for HTTP/HTTPS connections from the components in this Helm chart.
+  hosts: []
+
+  # The Serf LAN port that the clients will use to join the server cluster.
+  serfLANPort: 8301
+
+  # The HTTPS port of the Consul servers.
+  httpsPort: 443
```

Comment:
Really this should default to 8501.

Reply:
Will change this to 8501 for consistency.